### PR TITLE
Tidied AddPostFragment code, fixed filterspinner bug

### PR DESCRIPTION
**Pull request by LefanTan**
FIlterspinner previously shows words in big capital letters, fixed now.

### CODE REVIEW

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/SpinnerArrayAdapter.java [hunk 1]
```diff
@@ -66,22 +71,35 @@ public View getSpinnerView_DropDown(int position, View convertView, ViewGroup pa
         /*
         if None is the object, we can't use Mood to get the information because Mood doesn't have this moodType
          */
-        if(objects[position].equals("None")){
-            dropDownText.setText("None");
+        if(objects[position].equals( context.getResources().getString(R.string.filter_empty) )){
```
Comment (liushuyu) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/SpinnerArrayAdapter.java]: Unintended whitespace change?

```suggestion
        if(objects[position].equals(context.getResources().getString(R.string.filter_empty) )){
```

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/SpinnerArrayAdapter.java [hunk 2]
```diff
@@ -66,22 +71,35 @@ public View getSpinnerView_DropDown(int position, View convertView, ViewGroup pa
...
              */
             String moodName = objects[position].toUpperCase();
             Integer moodType = Mood.MoodTypes.get(moodName);
-            Mood mood = new Mood(moodType);
+            Mood mood = null;
+
+            try {
+                 mood = new Mood(moodType);
+            }catch (NullPointerException e){
+                Log.d("Error","Nullpointer exception for Filter " + e);
```
Comment (liushuyu) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/SpinnerArrayAdapter.java]: Use a more severe log level here:

```suggestion
                Log.e("FeelsLog", "Exception in Filter ", e);
```

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/SpinnerArrayAdapter.java [hunk 3]
```diff
@@ -66,22 +71,35 @@ public View getSpinnerView_DropDown(int position, View convertView, ViewGroup pa
...
+            }finally {
```
Comment (liushuyu) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/SpinnerArrayAdapter.java]: Whitespace problem.

```suggestion
            } finally {
```

**Review by liushuyu — CHANGES_REQUESTED** [review]
There are many whitespace related issues in this PR. You can make Android Studio or IDEA to automatically format the code for you in `Code` -> `Reformat Code`.

**Comment by LefanTan** [comment]
> There are many whitespace related issues in this PR. You can make Android Studio or IDEA to automatically format the code for you in `Code` -> `Reformat Code`.

Will do, thank you.

**Review by liushuyu — APPROVED** [review]
LGTM
Android tests are passed

**Pull request merged**